deep-learning/_posts/2018-01-16-3D-FCNN.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: review
+title:  "Hyperspectral Image Spatial Super-Resolution via 3D Full Convolutional Neural Network"
+tags:   hyperspectral
```

Comment:
Add more meaningful tags.  See table https://vitalab.github.io/howto/.  Also, add "hyperspectral" to the table.